website/docs/guides/0.8.2-upgrade-guide.html.markdown

```diff
@@ -0,0 +1,59 @@
+## Issue [INTMDB-175](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/398)
+### resources created with the affected versions ***v0.8.0/v0.8.1***
+
+The following resources were affected, due to a schema update,
+to achieve consistency for multiple providers (AWS) ()[]
+
+```
+mongodbatlas_privatelink_endpoint
+mongodbatlas_privatelink_endpoint_service
+```
+
+### Known issue from v0.8.0 to v0.8.2
+
+If the resources were created from v0.8.0, there was no error in the creation.
+However to upgrade to v0.8.2 the state sync steps must be done, otherwise the following error will be shown
+
+```
+Error: error reading MongoDB Private Service Endpoint Connection(601881cb0d97874a0eb4eb74): GET https://cloud.mongodb.com/api/atlas/v1.0/groups/5cf5a45a9ccf6400e60981b6/privateEndpoint/AZURE/endpointService//subscriptions/{SUSCRIPTION-ID}/endpoint/601881cb0d97874a0eb4eb74: 404 (request "RESOURCE_NOT_FOUND") Cannot find resource /api/atlas/v1.0/groups/5cf5a45a9ccf6400e60981b6/privateEndpoint/AZURE/endpointService//subscriptions{SUSCRIPTION-ID}/endpoint/601881cb0d97874a0eb4eb74.
+```
+
+### Known issue from v0.8.1 to v0.8.2
+
+In terraform v0.8.1 the resource creation is successful in the backend, but fails with the error of the read function.
+
+```
+Error: error adding MongoDB Private Service Endpoint Connection(/subscriptions/{SUSCRIPTION-ID}) to a Private Endpoint (6018680618d74a4b62fe0600): unexpected state '', wanted target 'AVAILABLE, REJECTED, DELETED'. last error: %!s(<nil>)
+
+```
+
+### State sync v0.8.2
```

Comment:
### Steps impacted users must perform to successfully upgrade to v0.8.2 with a correct state